suc/pkg/cmd.go

```diff
@@ -0,0 +1,64 @@
+package pkg
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/rancher/wins/suc/pkg/rancher"
+	"github.com/rancher/wins/suc/pkg/service"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+func Run(_ *cli.Context) error {
+	var errs []error
+	refreshService := false
+
+	initialState, err := service.BuildInitialState()
+	if err != nil {
+		return fmt.Errorf("could not build initial state for rancher-wins: %v", err)
```

Comment:
The same suggestion to all occurrences: 
```suggestion
		return fmt.Errorf("could not build initial state for rancher-wins: %w", err)
```